### PR TITLE
Add Sauce Connect page for Orchestrate.

### PR DESCRIPTION
**Pull request by kmissoumi**
### Description

Adding a page on Sauce Connect for Orchestrate.

### Motivation and Context

If the Orchestrate runner needs access to backend services, then an Orchestrate-compatible tunnel is needed. e.g. Create a test account via API, and query a database to use as test data.

### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/orchestrate/sauce-connect.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+title: Using Sauce Connect Tunnels with Sauce Orchestrate
+sidebar_label: Using Sauce Connect
+description: Get a Sauce Connect tunnel up and running quickly for your Sauce Orchestrate tests.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Sauce Orchestrate fully integrates with Sauce Connect. If the service or site you are testing is already accessed via Sauce Connect, no change to your code is needed.
```
Comment (LoryApiFortress) [inline docs/orchestrate/sauce-connect.md:12]:
```suggestion
Sauce Orchestrate fully integrates with Sauce Connect. If the service or site you are testing is already accessed via Sauce Connect, no change to your code is needed.
```

File: docs/orchestrate/sauce-connect.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import TabItem from '@theme/TabItem';
+
+Sauce Orchestrate fully integrates with Sauce Connect. If the service or site you are testing is already accessed via Sauce Connect, no change to your code is needed.  
+If your code makes direct calls to any backend service, such as an API or database that is behind a firewall on your private network, follow the steps below to launch a secure trusted connection between your network and the Sauce Orchestrate runner.
+
+
+## What You'll Need
+
+
+## Configure a Sauce Connect Tunnel with access to the management services.
```
Comment (LoryApiFortress) [inline docs/orchestrate/sauce-connect.md:19]:
```suggestion
## Configure a Sauce Connect Tunnel with Access to the Management Services
```

File: docs/orchestrate/sauce-connect.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+1.  The specific configuration needed for Sauce Orchestrate is setting the `vm-version` to the fixed value listed below and adding the domains of backend services to the [`tunnel-domains`](/dev/cli/sauce-connect-proxy/#--tunnel-domains) list.
+2.  All other steps and confgiration are the same. Please reference the [Sauce Connect Proxy Quickstart](/secure-connections/sauce-connect/quickstart/).
```
Comment (LoryApiFortress) [inline docs/orchestrate/sauce-connect.md:22]:
```suggestion
2.  All other steps and configuration are the same. Refer to the [Sauce Connect Proxy Quickstart](/secure-connections/sauce-connect/quickstart/). 
```

File: docs/orchestrate/sauce-connect.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+2.  All other steps and confgiration are the same. Please reference the [Sauce Connect Proxy Quickstart](/secure-connections/sauce-connect/quickstart/). 
+
+
+```yaml
+---
+vm-version: 'v2alpha2'
+tunnel-domains: [ ]
+```
+
+## Configure the Sauce Orchestrate runner to use this management tunnel.
```
Comment (LoryApiFortress) [inline docs/orchestrate/sauce-connect.md:31]:
```suggestion
## Configure the Sauce Orchestrate Runner to Use This Management Tunnel
```

File: docs/orchestrate/sauce-connect.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+Sauce Orchestrate fully integrates with Sauce Connect. If the service or site you are testing is already accessed via Sauce Connect, no change to your code is needed.  
+If your code makes direct calls to any backend service, such as an API or database that is behind a firewall on your private network, follow the steps below to launch a secure trusted connection between your network and the Sauce Orchestrate runner.
+
+
+## What You'll Need
+
+
+## Configure a Sauce Connect Tunnel with access to the management services.
+
+1.  The specific configuration needed for Sauce Orchestrate is setting the `vm-version` to the fixed value listed below and adding the domains of backend services to the [`tunnel-domains`](/dev/cli/sauce-connect-proxy/#--tunnel-domains) list.
```
Comment (LoryApiFortress) [inline docs/orchestrate/sauce-connect.md:21]: @kmissoumi the link is wrong, what are you trying to link to?

Comment (kmissoumi) [reply docs/orchestrate/sauce-connect.md:20]: https://docs.saucelabs.com/dev/cli/sauce-connect-proxy/#--tunnel-domains

File: docs/orchestrate/sauce-connect.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+```yaml
+---
+vm-version: 'v2alpha2'
+tunnel-domains: [ ]
+```
+
+## Configure the Sauce Orchestrate runner to use this management tunnel.
+
+1. Add the [`tunnel`](/secure-connections/orchestrate/saucectl-configuration/#tunnel) key to the runner configuration.
```
[the remark this answers is not in the document]
Comment (kmissoumi) [reply docs/orchestrate/sauce-connect.md:32]: https://docs.saucelabs.com/orchestrate/saucectl-configuration/#tunnel

**Comment by kmissoumi** [comment]
@LoryApiFortress, thank you for the review and suggestions!

File: docs/orchestrate/sauce-connect.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+```yaml
+---
+vm-version: 'v2alpha2'
+tunnel-domains: [ ]
+```
+
+## Configure the Sauce Orchestrate Runner to Use This Management Tunnel
+
+1. Add the [`tunnel`](/secure-connections/orchestrate/saucectl-configuration/#tunnel) key to the runner configuration.
```
Comment (LoryApiFortress) [inline docs/orchestrate/sauce-connect.md:32]:
```suggestion
1. Add the [`tunnel`](/orchestrate/saucectl-configuration/#tunnel) key to the runner configuration.
```

**Review by LoryApiFortress — APPROVED** [review]

**Pull request merged**